web_client/views/body/ImageView.js

```diff
@@ -571,6 +600,30 @@ var ImageView = View.extend({
 
     _setAnnotationOpacity(opacity) {
         this.viewerWidget.setGlobalAnnotationOpacity(opacity);
+    },
+
+    _rightClickElement(element, annotation, evt) {
+        if (annotation.get('_accessLevel') < AccessType.WRITE) {
+            return;
+        }
+
+        const menu = this.$('#h-annotation-context-menu');
```

Comment:
The browser context menu hides the custom context menu for me.  Don't we need to prevent the event default to avoid this?  If so, geojs probably needs to include the original event in its event.

Reply:
That's strange... I thought geojs was calling prevent default.

Reply:
@manthey I figured out that I wasn't getting the menu on my laptop because I wasn't logged in.  I still can't reproduce the issue of the default context menu appearing.  I've tried on several browsers on Linux and MacOS.